src/Table.luna

```diff
@@ -634,6 +968,25 @@ class Table:
     def fillNaAt columnName valueToFillWith:
         self.fromWrapper $ self.ptr.fillNaAt columnName valueToFillWith
 
+    # Linear interpolation of missing values in table.
```

Comment:
The style needs to be unified — do we open with third-person description ("interpolates"), imperative ("interpolate") or nominal sentence ("interpolation"). 
It is non-uniform across methods and even in a single method the are discrepancies.